tests/loadtests/vkloadtests/VulkanLoadTestSample.h

```diff
@@ -51,6 +65,16 @@ class VulkanLoadTestSample : public LoadTestSample {
                                     const std::string sBasePath);
 
   protected:
+
+    ktxVulkanTexture_subAllocatorCallbacks callbacks = {
```

Comment:
`callbacks` is a too generic a name. I'm struggling to come up with anything better.  My first thought was `vmaCallbacks` but in theory other suballocators could also be used, right?

Reply:
Yes, that is correct. Just called it `subAllocatorCallbacks`.